### PR TITLE
when clicked rightAnswer turns to green after two seconds

### PR DESCRIPTION
**Pull request by EnsarDogan**

### CODE REVIEW

**Comment by EnsarDogan** [comment]
when clicked rightAnswer turns to green after two seconds

**Comment by EnsarDogan** [comment]
when clicked rightAnswer turns to green after two seconds

File: src/views/rightAnswerView.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import {ANSWERS_LIST_ID} from '../constants.js';
+
+export const ulElement = document.getElementById(ANSWERS_LIST_ID)
+
+
```
Comment (ayktldg) [inline src/views/rightAnswerView.js]: I think this block is not used.

File: public/style.css [hunk 2]
```diff
@@ -66,4 +66,8 @@ ul {
 }
 .disabled{
   background-color: lightslategrey;
+}
+
+.pointer-none {
+  pointer-events: none;
 }
```
Comment (ayktldg) [inline public/style.css]: You can add a new line here otherwise, there could be merge conflicts

**Review by ayktldg — CHANGES_REQUESTED** [review]

**Review by ayktldg — APPROVED** [review]

**Pull request merged**